### PR TITLE
Bug: fix celo balance

### PR DESCRIPTION
**Pull request by L03TJ3**
# Description
Re-add G$Balance with fixed usd calculation
About # (link your issue here)
#337

### CODE REVIEW

**Comment by L03TJ3** [comment]
Not sure why vercel build fails, cannot replicate locally

File: src/pages/gd/Claim/ClaimBalance.tsx [hunk 1]
```diff
@@ -35,13 +33,14 @@ const ClaimTimer = () => {
...
-    //     () =>
-    //         g$Price()
-    //             .then(({ DAI }) => DAI)
-    //             .catch(() => undefined),
-    //     [chainId]
-    // )
+    const [G$Price] = usePromise(
+        () =>
+            g$Price()
+                .then(({ DAI }) => +DAI.toSignificant(6))
```
Comment (sirpy) [inline src/pages/gd/Claim/ClaimBalance.tsx:39]: what type is DAI? (ie original G$Price)
why aren't you the original G$Price to balancegd and let balancegd handle the display?

**Comment by sirpy** [comment]
> Not sure why vercel build fails, cannot replicate locally

try to remove node_modules and install and build good-design again

File: src/pages/gd/Claim/ClaimBalance.tsx (hunk 1 above)
Comment (L03TJ3) [reply src/pages/gd/Claim/ClaimBalance.tsx:39]: Dai is fraction so here I convert it .toSignifcant returns bigNumber
the + in front makes it a number

**Comment by L03TJ3** [comment]
> > Not sure why vercel build fails, cannot replicate locally
> 
> try to remove node_modules and install and build good-design again. make sure you re-release and point to it

Same issue, deleted cache + node_modules, run yarn install && yarn build
Locally it works?

File: src/pages/gd/Claim/ClaimBalance.tsx (hunk 1 above)
Comment (L03TJ3) [reply src/pages/gd/Claim/ClaimBalance.tsx:39]: @sirpy is this a suggestion or a 'should be done' comment?
I just think that passing down number is easier, and causes least amount of inconsistencies and/or issues

Comment (sirpy) [reply src/pages/gd/Claim/ClaimBalance.tsx:38]: It's a suggestion. The last rendering component should be in charge of the display

**Pull request merged**